src/entities/createPool/createPoolV3/liquidityBootstrapping/createLiquidityBootstrapping.ts

```diff
@@ -0,0 +1,40 @@
+import { getRandomBytes32 } from '@/entities/utils/getRandomBytes32';
+import { encodeFunctionData } from 'viem';
+
+import {
+    CreatePoolBase,
+    CreatePoolBuildCallOutput,
+    CreatePoolLiquidityBootstrappingInput,
+} from '../../types';
+
+import { lBPoolFactoryAbi_V3 } from '@/abi';
```

Comment:
Can you change to use "extended" abi - check out the other implementations as an example.

Reply:
addressed